src/util/unaligned.h

```diff
@@ -40,19 +40,25 @@
 /// Perform an unaligned 16-bit load
 static really_inline
 u16 unaligned_load_u16(const void *ptr) {
+    if (ptr == NULL) {
```

Comment:
same as other comments, this is in critical loop, add an assert for debug and a NOLINT.